### PR TITLE
2480: Reset user lists

### PR DESCRIPTION
**Pull request by BirkAndMe**
https://platform.dandigbib.org/issues/2480

### CODE REVIEW

File: modules/p2/ding_list/ding_list.module [hunk 1]
```diff
@@ -1196,3 +1196,10 @@ function ding_list_panels_pane_content_alter($content, $pane, $args, $contexts)
             array_slice($items, count($items) - 2, count($items) - 1, true);
   }
 }
+
+/**
+ * Implements hook_user_delete()
```
Comment (JrgenGNielsen) [inline modules/p2/ding_list/ding_list.module]: Scrutinizer: Manglende punktum

**Comment by JrgenGNielsen** [comment]
Please fix the Scrutinizer errors and the merge conflict. 
The 'Array indentation error' is probably a bug in Scrutinizer, though.

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Comment by kasperg** [comment]
Closed in favour of #902.

**Pull request closed without merging**